core/tauri/src/manager.rs

```diff
@@ -343,6 +343,15 @@ impl<P: Params> WindowManager<P> {
         current_window_label = label.to_js_string()?,
       ));
 
+    webview_attributes.uri_scheme_protocols.insert(
```

Comment:
Could we have a setter or something to update the path? 
Let say a user update change the path via a UI, asking a restart is a bit overkill I think?

Reply:
which path? the protocol just loads `asset:///path/to/file`.

Reply:
ho sorry, just saw that it's the dev responsibility to map the right path!
alls good